wpilibj/src/athena/java/edu/wpi/first/wpilibj/AnalogAccelerometer.java

```diff
@@ -58,10 +59,9 @@ public AnalogAccelerometer(final int channel) {
    *                connected to
    */
   public AnalogAccelerometer(AnalogInput channel) {
+    requireNonNull(channel, "Analog Channel given was null");
```

Comment:
I like the first option better. `requireNonNull()` is like an assert. asserts don't return things, and the second option violates that pattern.